### PR TITLE
fix(lambda-event-sources): add missing export of ./streams.ts

### LINKED ISSUE #4352: Stream has no exports
tsc fails to compile, StreamEventSource, StreamEventSourceProps missing from aws-lambda-event-sources streams file.

### Reproduction Steps
``` ts
npm install @aws-cdk/core
npm install @aws-cdk/aws-lambda-event-sources
tsc
```

### Error Log
``` bash
node_modules/@aws-cdk/aws-lambda-event-sources/lib/dynamodb.d.ts:3:10 - error TS2305: Module '"./stream"' has no exported member 'StreamEventSource'.

3 import { StreamEventSource, StreamEventSourceProps } from './stream';
           ~~~~~~~~~~~~~~~~~

node_modules/@aws-cdk/aws-lambda-event-sources/lib/dynamodb.d.ts:3:29 - error TS2305: Module '"./stream"' has no exported member 'StreamEventSourceProps'.

3 import { StreamEventSource, StreamEventSourceProps } from './stream';
                              ~~~~~~~~~~~~~~~~~~~~~~

node_modules/@aws-cdk/aws-lambda-event-sources/lib/kinesis.d.ts:3:10 - error TS2305: Module '"./stream"' has no exported member 'StreamEventSource'.

3 import { StreamEventSource, StreamEventSourceProps } from './stream';
           ~~~~~~~~~~~~~~~~~

node_modules/@aws-cdk/aws-lambda-event-sources/lib/kinesis.d.ts:3:29 - error TS2305: Module '"./stream"' has no exported member 'StreamEventSourceProps'.

3 import { StreamEventSource, StreamEventSourceProps } from './stream';
                              ~~~~~~~~~~~~~~~~~~~~~~
```
### Environment

  - **CLI Version      :10.
  - **Framework Version: 10.1.
  - **OS               :MacOS
  - **Language         : TypeScript

### Other
Digging around the node_modules/@aws-cdk/aws-lambda-event-sources/lib/stream.d.ts file shows no exports

--- 
This is :bug: Bug Report

### PR DESCRIPTION
**Pull request by RomainMuller**
Fixes #4352

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-lambda-event-sources/lib/stream.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import lambda = require('@aws-cdk/aws-lambda');
 import {Duration} from '@aws-cdk/core';
 
 /**
- * @internal
+ * @experimental
```
Comment (nija-at) [inline packages/@aws-cdk/aws-lambda-event-sources/lib/stream.ts]: Do we need the experimental tag, now that we're exporting it?

**Comment by aukaurea** [comment]
Hello @RomainMuller 

How are you?

Can you please help me know an ETA for this branch's release to npm?

Thank you.

**Comment by nija-at** [comment]
@aukaurea - We're working on releasing this within the next 2 days.

**Comment by aukaurea** [comment]
Thank yiou @nija-at

**Pull request merged**